### PR TITLE
feat: simple uniswap connector on arbitrum

### PR DESCRIPTION
**Pull request by pradyuman-verma**
tries resolving #113

### CODE REVIEW

File: contracts/arbitrum/connectors/uniswap-sell-beta/main.sol [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    function sell(
+        address tokenIn,
+        address tokenOut,
+        uint24 fee,
+        uint256 amountIn,
+        uint256 amountOutMinimum,
+        bool zeroForOne
+    ) public returns (uint256 amountOut) {
+        approveTransfer(tokenIn, msg.sender, address(this), amountIn);
+        amountOut = getSingleInput(
```
Comment (KaymasJain) [inline contracts/arbitrum/connectors/uniswap-sell-beta/main.sol]: Is this where the swap is happening? If so then never keep the function as `get` as that is mainly referred to a function that just returns some data without updating any storage.

Comment (KaymasJain) [reply contracts/arbitrum/connectors/uniswap-sell-beta/main.sol]: You can maybe rename it to - `swapSingleInput`

Comment (pradyuman-verma) [reply contracts/arbitrum/connectors/uniswap-sell-beta/main.sol]: ok sure

File: contracts/arbitrum/connectors/uniswap-sell-beta/main.sol [hunk 2]
```diff
@@ -8,25 +8,24 @@ abstract contract uniswapSellBeta is Helpers {
...
+    ) public payable returns (uint256 amountOut) {
+        approveTransfer(tokenIn, msg.sender, address(this), msg.value);
+        amountOut = swapSingleInput(
             getParams(
                 tokenIn,
                 tokenOut,
                 msg.sender,
                 fee,
-                amountIn,
+                msg.value,
```
Comment (KaymasJain) [inline contracts/arbitrum/connectors/uniswap-sell-beta/main.sol]: Why `msg.value`? You'll be swapping via WETH not ETH.

**Comment by pradyuman-verma** [comment]
connector deployed at - [0x313fe505ad3ead0d35dd5d6687fb9c6b2469db3d](https://arbiscan.io/address/0x313fe505ad3ead0d35dd5d6687fb9c6b2469db3d)

**Comment by KaymasJain** [comment]
Enabled connector - https://arbiscan.io/tx/0xf018440d65f438e6936597dead911ee6526f4e95f2b3edfaed820339ddfe0489

**Review by KaymasJain — APPROVED** [review]

**Pull request merged**